code/game/objects/machinery/computer/droppod_control.dm

```diff
@@ -32,8 +32,8 @@
 				var/obj/structure/droppod/pod = p
 				if(!length(pod.buckled_mobs))
 					continue
-				var/predroptime = rand(5, 1 SECONDS)	//Randomize it a bit so its staggered
-				addtimer(CALLBACK(pod, TYPE_PROC_REF(/obj/structure/droppod, launchpod), pod.buckled_mobs[1]), predroptime)
+				var/predroptime = rand(3, 1 SECONDS)	//Randomize it a bit so its staggered
+				addtimer(CALLBACK(pod, TYPE_PROC_REF(/obj/structure/droppod, start_launch_pod), pod.buckled_mobs[1]), predroptime)
```

Comment:
????

Reply:
What? Pod drops where changed slightly due to the new animations, this is just aligning the console drop with the change.